test/engine/SpatialJoinTest.cpp

```diff
@@ -1567,7 +1443,8 @@ TEST(SpatialJoin, getDescriptor) {
   SpatialJoin* spatialJoin = static_cast<SpatialJoin*>(op.get());
 
   auto description = spatialJoin->getDescriptor();
-  ASSERT_TRUE(description.find(std::to_string(spatialJoin->getMaxDist())) !=
+  ASSERT_TRUE(description.find(
```

Comment:
The better way (readability + diagnostics in failure case) is
`ASSERT_THAT(description, ::testing::HasSubstr("The substring that you want" ))`

(There is some opportunities to use this further below.)
(you can put in a local `using namespace ::testing` if you want)

Reply:
I did not change this piece of the code. Seems it was automatically changed by the code formatter.